src/backend/libs/efiling-cso-starter/src/test/java/ca/bc/gov/open/jag/efilingcsostarter/csoSubmissionServiceImpl/SubmitFilingPackageTest.java

```diff
@@ -173,13 +178,68 @@ public void testWithPopulatedSubmissionId() throws DatatypeConfigurationExceptio
                                 .statutoryFeeAmount(STATUTORY_FEE_AMOUNT)
                                 .create()))
                         .create(),
-                APP_CODE,
-                false,
                 efilingPaymentServiceMock);
 
         Mockito.verify(filingFacadeBeanMock, Mockito.times(1))
                 .submitFiling(ArgumentMatchers.argThat(filingPackage ->
+                        filingPackage.getProcRequest() == null &&
+                        filingPackage.getApplicationCd().equals(APP_CODE) &&
                         filingPackage.getCourtFileNo().equals(FILE_NUMBER) &&
+                        filingPackage.getLdcxCourtClassCd().equals(COURT_CLASS) &&
+                        filingPackage.getLdcxCourtDivisionCd().equals(DIVISION) &&
+                        filingPackage.getLdcxCourtLevelCd().equals(LEVEL) &&
+                        filingPackage.getSubmittedToAgenId().equals(AGENCY_ID) &&
+                        filingPackage.getDocuments().get(0).getPayments().get(0).getStatutoryFeeAmt().equals(STATUTORY_FEE_AMOUNT)));
+
+        Assertions.assertEquals(BigDecimal.TEN, actual.getTransactionId());
+        Assertions.assertEquals("http://cso/cso/accounts/bceidNotification.do?packageNo=10", actual.getPackageLink());
```

Comment:
this needs to be fixed... maybe not in this pr but soon (shouldnt have two `/cso`)